IAteIt/IAteIt/View/Feed/Component/FeedHeaderView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  PhotoCardView.swift
+//  IAteIt
+//
+//  Created by Beone on 2023/03/07.
+//
+
+import SwiftUI
+
+struct FeedHeaderView: View {
+    
+    let profilePicSize: CGFloat = 36
+    var PostInfo: TempFeedHeader
+    var tempPlateList = TempPlateData().list
+    
+    var body: some View {
+        VStack {
+            HStack(alignment: .top) {
+                ZStack {
+                    Rectangle()
+                        .aspectRatio(contentMode: .fit)
+                        .frame(width: profilePicSize, height: profilePicSize)
+                    Image(PostInfo.profileImage)
+                        .resizable()
+                        .scaledToFill()
+                        .layoutPriority(-1)
+                        .frame(width: profilePicSize, height: profilePicSize)
+                }
+                .clipped()
+                .cornerRadius(profilePicSize/2)
+                .padding([.top], 3)
+                
+                VStack(alignment: .leading) {
+                    Text(PostInfo.nickname)
+                        .font(.subheadline)
+                        .fontWeight(.semibold)
+                    HStack(alignment: .bottom, spacing: 8) {
+                        Text(PostInfo.place)
+                            .font(.footnote)
+                            .foregroundColor(.gray)
+                        Text("·")
+                            .font(.footnote)
+                            .foregroundColor(.gray)
+                        Text(PostInfo.time)
+                            .font(.footnote)
+                            .foregroundColor(.gray)
```

Comment:
```suggestion
                        Text("\(PostInfo.place) • \(PostInfo.time)")
                            .font(.footnote)
```

이렇게 하면 더 간결할 것 같습니다! 또 figma 기준으로 색상 gray는 안넣어도 될 것 같습니다^^;

Reply:
수정했습니다 감사합니다!